### PR TITLE
doc: add minutes for meeting 31 March 2022

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: meetings/2022-03-31.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* CPC update - no updates this week. Elections are coming up soon.
+* Board meeting - Nothing of note this week.
+
+### nodejs/node
+
+* export v8 trace API [#42462](https://github.com/nodejs/node/pull/42462)
+  * James asked that we defer this to a meeting that he can make.
```
Comment (richardlau) [inline meetings/2022-03-31.md:41]: Sorry, I got confused with this one and the "Publishing specs for `Buffer` and `EventEmitter`" issue, which is the one James asked to defer, not this one.

File: meetings/2022-03-31.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+*Richard Lau @richardlau (TSC)
+*Robert Nagy @ronag (TSC)
+*Tobias Nießen @tniessen (TSC)
+*Rich Trott @Trott (TSC)
+
+## Agenda
+
+### Announcements
+
+* Richard, release proposal for final 12.x release. Also reminder that it goes EOL
```
Comment (richardlau) [inline meetings/2022-03-31.md:27]:
```suggestion
* Richard, release proposal for final 12.x release, v12.22.12, <https://github.com/nodejs/node/pull/42531> with release candidate binaries available. Also reminder that it goes EOL
```

File: meetings/2022-03-31.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  * Richard, if we are not going to allow userland modules with the same namespace
+    then what is the point.
+  * Antoine, long term plan would be to have all modules under the `node:` scheme
+    but don’t need to force that.
+  * Michael, since it can happen already due to namespaces, how much risk does it
+    add?
+  * Richard, could we add a flag which prevented it.
+  * Antoine, if we use `node:` scheme is breaking change as it required changes
+    in test scheme.
+  * Rich lets take back to the issue tracker, and commit to make a decision in the
```
Comment (Trott) [inline meetings/2022-03-31.md:66]:
```suggestion
  * Rich: Let's take this back to the issue tracker, and commit to make a decision in the
```

File: meetings/2022-03-31.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    then what is the point.
+  * Antoine, long term plan would be to have all modules under the `node:` scheme
+    but don’t need to force that.
+  * Michael, since it can happen already due to namespaces, how much risk does it
+    add?
+  * Richard, could we add a flag which prevented it.
+  * Antoine, if we use `node:` scheme is breaking change as it required changes
+    in test scheme.
+  * Rich: Let's take this back to the issue tracker, and commit to make a decision in the
+    Next 7 days.
```
Comment (Trott) [inline meetings/2022-03-31.md:67]:
```suggestion
    next 7 days.
```

**Review by Trott — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

**Review by RaisinTen — APPROVED** [review]

**Pull request merged**